tests/assert/assert.go

```diff
@@ -21,3 +21,13 @@ func Error(t *testing.T, err error, expected string) {
 		t.Fatalf("expected '%s' but got '%s'", expected, err.Error())
 	}
 }
+
+// True asserts that an interface is equal to boolean "true"
+func True(t *testing.T, expected interface{}) {
```

Comment:
For the record, the answer to when we might adopt testify at the moment is "when they fix their dependency chain." They've vendored their own dependencies even though they're a library, so I'm really not interested into wading into that mess right now (and `dep` is still in alpha)